specification/datadog/resource-manager/Microsoft.Datadog/stable/2022-08-01/datadog.json

```diff
@@ -0,0 +1,2013 @@
+        "createdBy": {
+          "description": "The user that created the API key.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the API key.",
+          "type": "string"
+        },
+        "key": {
+          "description": "The value of the API key.",
+          "type": "string"
+        },
+        "created": {
+          "description": "The time of creation of the API key.",
+          "type": "string"
+        }
+      }
+    },
+    "DatadogApiKeyListResponse": {
+      "description": "Response of a list operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Results of a list operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DatadogApiKey"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next set of results, if any.",
+          "type": "string"
+        }
+      }
+    },
+    "DatadogInstallMethod": {
+      "type": "object",
+      "properties": {
+        "tool": {
+          "description": "The tool.",
+          "type": "string"
+        },
+        "toolVersion": {
+          "description": "The tool version.",
+          "type": "string"
+        },
+        "installerVersion": {
+          "description": "The installer version.",
+          "type": "string"
+        }
+      }
+    },
+    "DatadogLogsAgent": {
+      "type": "object",
+      "properties": {
+        "transport": {
+          "description": "The transport.",
+          "type": "string"
+        }
+      }
+    },
+    "DatadogHostMetadata": {
+      "type": "object",
+      "properties": {
+        "agentVersion": {
+          "description": "The agent version.",
+          "type": "string"
+        },
+        "installMethod": {
+          "$ref": "#/definitions/DatadogInstallMethod"
+        },
+        "logsAgent": {
+          "$ref": "#/definitions/DatadogLogsAgent"
+        }
+      }
+    },
+    "DatadogHost": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the host.",
+          "type": "string"
+        },
+        "aliases": {
+          "description": "The aliases for the host.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "apps": {
+          "description": "The Datadog integrations reporting metrics for the host.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "meta": {
+          "$ref": "#/definitions/DatadogHostMetadata"
+        }
+      }
+    },
+    "DatadogHostListResponse": {
+      "description": "Response of a list operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Results of a list operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DatadogHost"
```

Comment:
Consider to add x-ms-identifiers extension for array properties. [http://azure.github.io/autorest/extensions/#x-ms-identifiers](http://azure.github.io/autorest/extensions/#x-ms-identifiers)

Reply:
Point taken. Since this will be a breaking change, we'll raise a separate PR just to accommodate all the changes mentioned by you. Currently, the purpose of the PR is to accommodate multi-subscription changes.